users/miles2go/babblePaste.txt

```diff
@@ -0,0 +1,198 @@
+ BabblePaste is a universal translator for keyboard shortcuts.
```

Comment:
nit: I think this would be much more readable as a Markdown file.

Reply:
I agree. Fixed.